phpBB/includes/acp/acp_groups.php

```diff
@@ -985,6 +985,7 @@ function main($id, $mode)
 					'S_GROUP_SPECIAL'	=> ($row['group_type'] == GROUP_SPECIAL) ? true : false,
 
 					'GROUP_NAME'	=> $group_name,
+					'GROUP_COLOR'	=> $row['group_colour'],
```

Comment:
We use colour instead of color.

Reply:
Yeah, taken into account, but as far as I know, when you use this variable to assign CSS' `color` prop, then it should be `COLOR`. But if you really insist, I will rename it.